configs/ot2_defconfig

```diff
@@ -50,6 +50,7 @@ BR2_PACKAGE_PYTHON3_SQLITE=y
 BR2_PACKAGE_PYTHON3_XZ=y
 BR2_PACKAGE_PYTHON_AIONOTIFY=y
 BR2_PACKAGE_PYTHON_CHARDET=y
+BR2_PACKAGE_PYTHON_CURSES=y
```

Comment:
```suggestion
BR2_PACKAGE_PYTHON3_CURSES=y
```

As written, this will actually affect the python 2 (which we don't build in) build - BR2_PACKAGE_PYTHON_FOO invocations for actual separate python packages tend to be python version independent, but [BR2_PACKAGE_PYTHON_CURSES](https://github.com/Opentrons/buildroot/blob/b6cedf7c863e0dd0ce8de728b38169415b32d9d3/package/python/Config.in#L59) (and the one we should use,  [BR2_PACKAGE_PYTHON3_CURSES](https://github.com/Opentrons/buildroot/blob/b6cedf7c863e0dd0ce8de728b38169415b32d9d3/package/python3/Config.in#L55)) are actually options to the interpreter builds. It's annoying they're named the same.

Reply:
Thanks for catching this and sorry for making work for you folks, as you can see I have no real idea what I'm doing with buildroot.